portfolio/src/main/webapp/index.html

```diff
@@ -57,92 +61,104 @@ <h2>Aspiring Software Engineer</h2>
+        <div class="info-row">
+          <div class="container">
+            <div class="row">
+              <div class="about">
+                <center>
+                  <h2>My Favorites</h2>
+                  <p class="struct">
+                    Long Drives<br />
+                    Avocados<br />
+                    Lavendar<br />
+                    Sweater Weather<br />
+                  </p>
+                  </div>
+                </center>
+                <div class="image">
+                  <center><img class="img-responsive" src="images/favorites.png" height="250" width="250" /></center>
+                </div>
+              </div>
+            </div>
+          </div>
+          <!-- Map -->
+          <div id="map" class="map_format"></div>
+          <!-- Generate Random Quote -->
+          <div class="info-row info-row1">
             <center>
-              <h2>My Favorites</h2>
-              <!-- TODO: Change to unordered list & keep consistent formatting -->
               <p class="struct">
-                Long Drives<br>
-                Avocados<br>
-                Lavendar<br>
-                Sweater Weather<br>
+                <body onload="getRandomQuote()">
+                  <div id="quote-container"></div>
+                  </p>
+                  </div>
+                  </center>
+                  <!-- Comment Box-->
+                  <div class="container_comment">
+                    <form method="POST" action="/data">
+                      <center>
+                        <label for="fname">First Name</label>
+                        <input type="text" id="fname" name="firstname" placeholder="Your first name.." />
+                        <label for="lname">Last Name</label>
+                        <input type="text" id="lname" name="lastname" placeholder="Your last name.." />
+                        <label for="country">Country</label>
+                        <select id="country" name="country">
+                          <option value="USA">USA</option>
+                          <option value="Canada">Canada</option>
+                          <option value="Australia">Australia</option>
+                        </select>
+                        <label for="subject">Subject</label>
+                        <textarea
+                          id="subject"
+                          name="subject"
+                          placeholder="Write something.."
+                          style="height: 200px;"
+                        ></textarea>
+                        <input type="submit" value="Submit" />
+                      </center>
+                    </form>
+                  </div>
+                </body>
               </p>
-          </div>
-          </center>
-          <div class="image">
-            <center><img class="img-responsive" src="images/favorites.png" height="250" width="250"/></center>
+            </center>
           </div>
```

Comment:
The following tags apparently don't have a matching opening tag. Are they needed?